src/InitializePackagesCommandLineHandler/InitializePackagesCommandLineHandler.class.st

```diff
@@ -75,19 +75,19 @@ InitializePackagesCommandLineHandler >> initializePackagesFrom: aFileName [
 { #category : #activation }
 InitializePackagesCommandLineHandler >> initializeProtocolsFrom: aFileName [
 
-	(File named: aFileName) readStreamDo: [:binaryStream | | stream |
+	(File named: aFileName) readStreamDo: [ :binaryStream |
+		| stream |
 		stream := ZnCharacterReadStream on: binaryStream encoding: 'utf8'.
-		[stream atEnd] whileFalse: [ | line items selector class |
+		[ stream atEnd ] whileFalse: [
+			| line items selector class |
 			line := stream upTo: Character cr.
 			items := line findTokens: String tab.
-			class := (items second = 'true')
-				ifTrue: [(Smalltalk classOrTraitNamed: items first) classSide]
-				ifFalse: [Smalltalk classOrTraitNamed: items first].
+			class := items second = 'true'
+				         ifTrue: [ (Smalltalk classOrTraitNamed: items first) classSide ]
```

Comment:
Cyril I know that this is not you but could we try self class environment instead of Smalltalk?